frontend/src/Context/PreviewPublisherProvider/usePreviewPublisher/usePreviewPublisher.spec.tsx

```diff
@@ -69,15 +69,18 @@ describe('usePreviewPublisher', () => {
   });
 
   describe('initLocalPublisher', () => {
-    it('should call initPublisher', () => {
+    it('should call initPublisher', async () => {
       mockedInitPublisher.mockReturnValue(mockPublisher);
       const { result } = renderHook(() => usePreviewPublisher());
+
       result.current.initLocalPublisher();
 
-      expect(mockedInitPublisher).toHaveBeenCalled();
+      await waitFor(() => {
```

Comment:
Note: had to make these `waitFor` since inside `initLocalPublisher` we have an `await` now. Without `waitFor` the spy isn't called

Reply:
why not just `await` initLocalPublisher?

Reply:
oh yea that works, too. had to change some `act`s to `await` though but yea that works, thanks 🙌